Readme.md

```diff
@@ -70,7 +70,36 @@ $ echo '{"pid":94473,"hostname":"MacBook-Pro-3","level":30,"msg":"hello world","
 <134>1 2016-04-01T16:44:58Z MacBook-Pro-3 - 94473 - - {"pid":94473,"hostname":"MacBook-Pro-3","level":30,"msg":"hello world","time":1459529098958,"v":1}
 ```
 
-## Install
+
+## Usage as Pino Transport
+
+You can use this module as a [pino transport](https://getpino.io/#/docs/transports?id=v7-transports) like so:
+
+```js
+const pino = require('pino')
+const transport = pino.transport({
+  target: 'pino-syslog',
+  level: 'info',
+  options: {
+    destination: 1, // optional (default stdout)
+    ... // other options
+  }
+})
+pino(transport)
+```
```

Comment:
Can you please update these docs with the usage of pino-socket as well?

Reply:
Do we need to directly depend on and integrate with `pino-sockect`? Shouldn't it just be a documentation example with maybe an included test?

Reply:
> Can you please update these docs with the usage of pino-socket as well?

Sure

> Do we need to directly depend on and integrate with pino-sockect? Shouldn't it just be a documentation example with maybe an included test?

Yes, it is. the `pino-socket` is a dev-dep and you should be able to pipe the output to this module to other destinations (such mongodb)

Reply:
I think the main usecase is to use `pipeline`.